mkdocs.yml

```diff
@@ -25,6 +25,7 @@ nav:
       - Configuration: server/configuration.md
       - Auth: server/auth.md
   - Integrations:
+      - integrations/index.md
```

Comment:
Minor nit, could you update this to ` - Integrations/index.md` so its consistent with all of the other changes?

Reply:
I'm not sure I understand - do you mean capitalize the I of `integrations`? for all of the paths?

Reply:
Oh sorry for not clarifying, the request was only for line 28 so it matches the case for the rest of `mkdocs.yml`

Reply:
I've updated - lmk if this is what you had in mind